src/v/transform/tests/test_fixture.cc

```diff
@@ -72,9 +72,21 @@ ss::future<> fake_source::push_batch(model::record_batch batch) {
     co_return;
 }
 
-ss::future<> fake_wasm_engine::start() { return ss::now(); }
+ss::future<> fake_wasm_engine::start() {
+    if (_started) {
+        throw std::logic_error("starting already started wasm engine");
```

Comment:
Generally considered an anti-pattern that leads to issues to throw directly in a non-coroutine since it won't be wrapped in a future so composability is lost. Was that intended?

Reply:
No it was not. I can send up another PR to make it a coroutine.

Reply:
Oh wait this isn't merged, I'll update this here

Reply:
Updated, thanks - was being a bit sloppy.